### ISSUE TITLE
Create new protocols subdirectory

### ISSUE THREAD

**Issue by rgerkin** [scidash/neuronunit#75]
There should be a protocols file or directory of files containing protocols associated with Capabilities like current injection.  These protocols could be used by tests, and would exist at a level in between direct calls to capabilities and whole `compute_prediction` methods.  See also #71.  An example of two different protocols would be "inject a square wave of size X" and "inject a triangle wave of size Y" and "inject a series of short pulses of size Z".

**Comment by rgerkin** [comment]
@JustasB Did you ever make anything like this in your work?

**Comment by JustasB** [comment]
I agree, see my response here: https://github.com/scidash/neuronunit/issues/71#issuecomment-359188246

**Comment by rgerkin** [comment]
@JustasB Are you satisfied with the current implementation of this request in the dev branch here?  Is there anything you need to push to satisfy it?

**Comment by JustasB** [comment]
@rgerkin I have merged everything I had worked on thus far. I don't have any other outstanding code to push.

I think the separate protocols folder idea is something that we probably should do with any future tests, and we need to decide if it's feasible to expend the effort to refactor the existing tests.

**Comment by rgerkin** [comment]
If you have the protocols defined in an nmldb repo, could you point me to it, and to how you use that in the testing?  Then maybe I can do this refactoring.

**Comment by JustasB** [comment]
Take a look at these:

The AIBS and HBP (long square and input resistance) protocols are defined here: https://github.com/scrook/neuroml-db/blob/master/Import%20Scripts/model-importer/cellmodel.py

The general workflow is: 

a) each model is subjected to the AIBS and HBP protocols 
b) the input and output waveforms are saved to the nmldb (and shown on the nmldb website) 
c) a NU staticmodel defined [here](https://github.com/scidash/neuronunit/blob/dev/neuronunit/neuromldb.py) gets the waveforms from [nmldb API](https://neuroml-db.org/api), and uses them to compute the Druckmann properties
